src/house.cpp

```diff
@@ -190,37 +190,37 @@ bool House::transferToDepot() const
 		return false;
 	}
 
-	Player* player = g_game.getPlayerByGUID(owner);
+	std::shared_ptr<Player> player = g_game.getPlayerByGUID(owner);
 	if (player) {
 		transferToDepot(player);
 	} else {
-		Player tmpPlayer(nullptr);
-		if (!IOLoginData::loadPlayerById(&tmpPlayer, owner)) {
+		std::shared_ptr<Player> tmpPlayer = nullptr;
```

Comment:
Could this be `auto`
```suggestion
		auto tmpPlayer = std::shared_ptr<Player>(nullptr);
```